### DISCUSSION TITLE
Deliberate orphan pages?

### DISCUSSION THREAD

**Discussion by djr319** [parcel-bundler/parcel#7686] [Q&A] [answered]
Hi, does anybody have any experience in the following? 

I want a "secret" webpage (for a product discount, before your imagination goes too wild!).

- It is not linked to from the "main site" ... but would be landed on directly
- It should not be indexed by search engines (what is a sourceMap? Will this expose such pages?)
- It needs to be "built" by Parcel

Any ideas? I think that "tree-shaking" will also mean that HTML files are ignored if not linked too, is that correct? If so, do I just need to upload the page manually, or is there a way to ensure that Parcel builds the page? (without making it crawlable by search engines)

Sorry if some of this creeps outside of Parcel discussions strictly... appreciate any tips from you 🙏

**Comment by mischnic** [comment, marked as answer]
If you're asking how to make Parcel process a HTML file without it being linked to via `<a href>` from another HTML file: do `parcel build src/index.html src/secret.html`.

> It should not be indexed by search engines (what is a sourceMap? Will this expose such pages?)

Sourcemaps have nothing to do with search engine indexing

What you need can be done with a robots.txt file: https://developers.google.com/search/docs/advanced/robots/create-robots-txt

**Reply by djr319** [reply]
Hi mischnic. Many thanks for answering my questions 🙏
